docs/advance-toolchain-documentation.html

```diff
@@ -829,6 +861,16 @@
 <p><span style="font-family:courier new,courier,monospace;">/opt/atX.X/etc/ld.so.conf</span> already includes <span style="font-family:courier new,courier,monospace;">/etc/ld.so.conf</span> in the search order, but you may need to re-run <span style="font-family:courier new,courier,monospace;">/opt/atX.X/sbin/ldconfig</span> in order populate <span style="font-family:courier new,courier,monospace;">/opt/atX.X/etc/ld.so.cache</span> with the specialized search paths you have added to <span style="font-family:courier new,courier,monospace;">/etc/ld.so.conf</span> after an Advance Toolchain installation.</p>
 
 <p>If you are running <span style="font-family:courier new,courier,monospace;">ldd</span> against your binary and it is showing that some libraries are not found, you may need to re-run <span style="font-family:courier new,courier,monospace;">/opt/atX.X/sbin/ldconfig</span> to fix that.</p>
+
+<p>The environment variable LD_LIBRARY_PATH is a colon-separated set of directories where libraries should be searched first before the standard set of directories. This list is used in preference to any runtime or default system linker path.</p>
+<p>We do not recomend the use of LD_LIBRARY_PATH, you usually can rely on the list compiled in path, but if you have to use it (e.g. your build system use LD_LIBRARY_PATH or have it as part of a wrapper) be sure that a system directory <b>never appears before</b> the Advance Toolchain directory.</p>
```

Comment:
s/you usually can rely/you can usually rely/ (subtle: you want the adverb "usually" modifying "can" rather than "you". Either would be understood, though.
s/build system use LD_LIBRARY_PATH or have it/build system uses LD_LIBRARY_PATH or has it/